### PR TITLE
Add preseed file and new xinitrc to idiot

### PR DESCRIPTION
**Pull request by WillNilges**
Introduces a new way to set up Devcade devices that is _almost_ completely unattended. This file can be provided to a Debian ISO and will configure accounts, packages, services, Xorg, login, and applications automatically.

No more fuss.

### CODE REVIEW

**Comment by WillNilges** [comment]
Need to link the preseed in the website. I'm thinking about adding a thing in the Dockerfile that will copy the file into the website or something.

File: idiot/.env [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export VIEW_WIDTH=
-export VIEW_HEIGHT=
-export DEVCADE_API_DOMAIN=
-export DEVCADE_DEV_API_DOMAIN=
+export VIEW_WIDTH=1080
+export VIEW_HEIGHT=1920
```
Comment (MTFT-Games) [inline idiot/.env]: Should this not default to 2160 or whatever ultrawide is?

File: idiot/.env [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
+export DEVCADE_API_DOMAIN=devcade-api.apps.okd4.csh.rit.edu
+export DEVCADE_DEV_API_DOMAIN=devcade-api-dev.apps.okd4.csh.rit.edu
```
Comment (MTFT-Games) [inline idiot/.env:4]: should we theoretically leave these blank and ask the user to support the eventual goal of this being an open platform that anyone can set up their own instance of?

File: idiot/.xinitrc [hunk 3]
```diff
@@ -1,37 +1,16 @@
 #!/bin/sh
 
-# /etc/X11/xinit/xinitrc
-#
-# global xinitrc file, used by all X sessions started by xinit (startx)
+configure_display() {
+	RESOLUTION="$VIEW_WIDTH"'x'"$VIEW_HEIGHT"'_75.00'
```
Comment (MTFT-Games) [inline idiot/.xinitrc]: Is this resolution ever used?

File: idiot/.env (hunk 1 above)
Comment (WillNilges) [reply idiot/.env]: You're so right bestie

File: idiot/.env (hunk 2 above)
Comment (WillNilges) [reply idiot/.env:4]: Eventually, but for now, I think it's fine to leave them for convenience's sake.

File: idiot/.xinitrc (hunk 3 above)
Comment (WillNilges) [reply idiot/.xinitrc]: 'spose not. I was using it, but now I am not using it.

**Review by MTFT-Games — APPROVED** [review]
Cool

**Pull request merged**